apple/Sources/FerrostarSwiftUI/Views/SpeedLimit/USSpeedLimitView.swift

```diff
@@ -0,0 +1,74 @@
+import SwiftUI
+
+/// A US-style speed limit sign using a rectangular white box and a black border.
+public struct USStyleSpeedLimitView: View {
```

Comment:
Instead of "US" and "US-style", how about calling it "Imperial"?
```suggestion
public struct ImperialSpeedLimitView: View {
```

Reply:
`Imperial` describes a system of measurement units. But the same style of sign is used in Canada with metric units :P I tried to come up with some better ideas than "US-style" as a classifier, and I _think_ that MUTCD might be correct, but to be honest I had a hard time verifying this.

If anyone can verify that MUTCD is the right term, we can go with that (perhaps `MUTCDStyleSpeedLimitView`). Otherwise, "US-style" is a reasonably accurate descriptor.